### PR TITLE
kustomize: init at 1.0.4

### PR DESCRIPTION
**Pull request by carlosdagos**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/tools/kustomize/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  };
+
+  meta = with lib; {
+    description = "Customization of kubernetes YAML configurations";
+    longDescription = ''
+      kustomize lets you customize raw, template-free YAML files for
+      multiple purposes, leaving the original YAML untouched and usable
+      as is.
+    '';
+    homepage = "https://github.com/kubernetes-sigs/kustomize";
```
Comment (ryantm) [inline pkgs/development/tools/kustomize/default.nix]: URLs don't need quotes around them in Nix, can you remove them?

**Review by ryantm — CHANGES_REQUESTED** [review]

**Comment by ryantm** [comment]
@GrahamcOfBorg build kustomize

File: pkgs/development/tools/kustomize/default.nix (hunk 1 above)
Comment (carlosdagos) [reply pkgs/development/tools/kustomize/default.nix]: Didn't know that. Will do so 😄

**Comment by carlosdagos** [comment]
Changes done. Thanks for reviewing @ryantm.

**Comment by ryantm** [comment]
@GrahamcOfBorg build kustomize

**Review by ryantm — APPROVED** [review]

**Pull request merged**